### PR TITLE
Code example

### PR DESCRIPTION
**Pull request by MSBradshaw**
Created a python module for housing functions for interacting with scikit-learn models. Also added a main script for actually running analysis.

### CODE REVIEW

File: main.py [hunk 1]
```diff
@@ -0,0 +1,18 @@
+import Learner_Functions as fl
```
Comment (jdayton3) [inline main.py]: Why `fl` instead of `lf`?

File: Learner_Functions.py [hunk 2]
```diff
@@ -0,0 +1,26 @@
+"""
```
Comment (jdayton3) [inline Learner_Functions.py]: [PEP8 says](https://www.python.org/dev/peps/pep-0008/?#package-and-module-names) module names should be all lowercase.

File: Learner_Functions.py [hunk 3]
```diff
@@ -0,0 +1,26 @@
...
+from sklearn.neighbors import KNeighborsClassifier
+from sklearn.model_selection import cross_val_score, StratifiedShuffleSplit
+
+RAND_STATE = 0
+TEST_SIZE = 0.3
+NUMBER_OF_SPLITS = 10
+SCORING_METHOD = 'accuracy'
+
+"""
+
```
Comment (jdayton3) [inline Learner_Functions.py]: What's this empty docstring?

File: main.py [hunk 4]
```diff
@@ -0,0 +1,18 @@
...
+
+#read in the training files
+clinical = pd.read_csv('train_cli.tsv', sep='\t')
+protein = pd.read_csv('train_pro.tsv', sep='\t')
+
+#impute missing values with 0
+protein = protein.fillna(0)
+clinical = clinical.fillna(0)
+
+#remove the sample labels and transform the data such that protiens are features
```
Comment (jdayton3) [inline main.py]: "protien" -> "protein" :slightly_smiling_face:

File: main.py (hunk 1 above)
Comment (jdayton3) [inline main.py]: I get a couple of `DataConversionWarning`s when I run this file with Python 3:

```
/home/jdayton3/miniconda3/envs/py36/lib/python3.6/site-packages/sklearn/model_selection/_validation.py:458: DataConversionWarning: A column-vector y was passed when a 1d array was expected. Please change the shape of y to (n_samples, ), for example using ravel().
  estimator.fit(X_train, y_train, **fit_params)
...
```

^ Repeated 10x.

Comment (jdayton3) [inline main.py]: Would it make sense to remove this file and put the code in a `if __name__ == "__main__":` block in `Learner_Functions.py` instead?

**Review by jdayton3 — CHANGES_REQUESTED** [review]
Looks great! I think this will be super helpful for getting people oriented with scikit-learn. 

A few general comments:

* Can we document how to use these files, either in the `README.md` or in a `CONTRIBUTING.md`?
* It might be nice to have some pre-merge linting checks for PEP8 compliance, especially once we have ~10 different people committing code. Spyder also makes it easy to check that stuff (at least on Ubuntu, you just have to hit `F8` and it runs pylint on the current file). (This would probably be a separate pull request getting it set up.)
* ~~Should we make subdirectories for modules / packages, scripts, and data? It might make the repo a little more organized.~~ Edit: I did this in #4

**Comment by MSBradshaw** [comment]
Fixed spelling mistakes
added to the read me
added a missing column name to train_pro.tsv

**Review by jdayton3 — APPROVED** [review]

**Pull request merged**